### DISCUSSION TITLE
Dynamic menu: add my custom icons

### DISCUSSION THREAD

**Discussion by edgar-koster** [ueberdosis/tiptap#942] [General Questions]
My menu looks like this
![image](https://user-images.githubusercontent.com/24190040/104711894-38e57b00-5722-11eb-9f75-08573f5e710e.png)
When I click on  a table it changes to
![image](https://user-images.githubusercontent.com/24190040/104711930-43a01000-5722-11eb-8bf8-eb49b2c69283.png)
I love this feature, but where is this comming from for I want to change this layout

**Comment by edgar-koster** [comment]
I figured this out. It wasn't tiptap (but the idea is there too) but quasar-tiptap. I adjusted that so 'tableToolbar' is a property passed to the editor and you can then make a custom toolbar for the table.

**Comment by vesper8** [comment]
@edgar-koster Is there an open-source project that provides all these cool icons for common tiptap extensions, or did you build all that yourself?
